### ISSUE TITLE
load_experiment tests

### ISSUE THREAD

**Issue by meta00** [r-and-gama/rama#14]
load_experiment(exp_name, gaml_file)

1. Test if the name of exp_name exists in gaml_file
2. Test if gaml_file exists

**Comment by LucieContamin** [comment]
Fixed in commits: `7df776f` & `4db4a92`

**Comment by meta00** [comment]
create working directory, check if there exists a file with the same name

```r
 if(!file.exists(wk_dir))
    # Check if a file name dir exist already
    dir.create(wk_dir)
```
